WolvenKit.App/ViewModels/Tools/ChunkViewModelExtended/ChunkViewModel.SearchAndReplace.cs

```diff
@@ -32,17 +32,19 @@ public static void SearchAndReplace_ResetCaches()
     /// <param name="replace"></param>
     /// <param name="isWholeWord"></param>
     /// <param name="isRegex"></param>
+    /// <param name="ignoreCache"></param>
     /// <returns></returns>
-    private int SearchAndReplaceInternal(string search, string replace, bool isWholeWord, bool isRegex)
+    private int SearchAndReplaceInternal(string search, string replace, bool isWholeWord, bool isRegex,
+        bool ignoreCache = false)
     {
         NumReplacedEntries = 0;
         var isExpanded = IsExpanded;
 
         // SearchAndReplaceInProperties can influence the expansion state, even if nothing was changed
-        var hasReplacement = SearchAndReplaceInProperties(search, replace, isWholeWord, isRegex);
+        var hasReplacement = SearchAndReplaceInProperties(search, replace, isWholeWord, isRegex, ignoreCache);
         IsExpanded = isExpanded;
 
-        if (hasReplacement)
+        if (!hasReplacement)
         {
             return NumReplacedEntries;
```

Comment:
Where is this variable set?

Reply:
Yeah I know, but it's a bit of a bad design. the member variable just gets set in seemingly random places. We should remove it from the CVM tbh, and make it local